editor/editor_node.cpp

```diff
@@ -4636,6 +4636,18 @@ void EditorNode::_scene_tab_changed(int p_tab) {
 	editor_data.get_undo_redo().commit_action();
 }
 
+String EditorNode::capitalize_string(const String &p_string) const {
+
+	String capitalized_string = p_string.capitalize();
```

Comment:
Is there a case where two different strings could result in the same capitalized string? If the answer is yes, then the current implementation is understandable. If not, then this can be optimized by only calling `capitalize()` if a remap could not be found below.

Reply:
I agree, and this would remove the awkwardness of having remappings based on the wrongly capitalized property paths (which can change if we ever change what `capitalize()` does).

Instead, remap directly based on the property path substrings:
```
capitalize_string_remaps["s3tc"] = "S3TC";
```